ray-on-gke/platform/modules/gke_standard/main.tf

```diff
@@ -87,3 +96,22 @@ resource "google_container_node_pool" "gpu_pool" {
     }
   }
 }
+
+resource "google_container_node_pool" "single_host_tpu_pool" {
```

Comment:
Can we just call it "tpu_pool" for simplicity? The user can actually change it to multihost pods if they want.